proarc-webapp/src/main/java/cz/cas/lib/proarc/webapp/server/rest/DigitalObjectResource.java

```diff
@@ -213,7 +225,20 @@ public DigitalObjectResource(
                         DigitalObjectResourceApi.DIGITALOBJECT_PID, "Invalid PID!").build();
             }
         }
-        xmlMetadata = (xmlMetadata == null || xmlMetadata.isEmpty() || "null".equals(xmlMetadata)) ? null : xmlMetadata;
+
+        if (wfJobid != null) {
```

Comment:
Asi bych propojení DO a workflow přesunul do DigitalObjectManager. Jednak proarc-webapp je spíš transportní vrstva a jinak mi to přijde jako další vlastnost DO, tak jako další RELS-EXT relace.